### PR TITLE
Navbar fix

### PR DESCRIPTION
**Pull request by BastienSaulnier**

### CODE REVIEW

**Review by LeoDiduch — DISMISSED** [review]
SnE

File: frontend/src/App.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import NavBar from "./components/NavBar";
```
Comment (darkweak) [inline frontend/src/App.js:2]: named import instead

File: frontend/src/components/NavBar.jsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+import React, { useState } from "react";
 import { scaleRotate as BurgerMenu } from "react-burger-menu";
 import { NavLink } from "react-router-dom";
 import { useDispatch } from "react-redux";
 import "./styles/NavBar.scss";
 
 function NavBar() {
   const dispatch = useDispatch();
+  const [isBurgerOpen, isBurgerOpenUpdate] = useState(false);
+  const isMenuOpen = state => {
```
Comment (darkweak) [inline frontend/src/components/NavBar.jsx:10]:
```suggestion:0+2
  const isMenuOpen = { isOpen } => {
    isBurgerOpenUpdate(isOpen);
    return isOpen;
  }
```

File: frontend/src/components/NavBar.jsx [hunk 3]
```diff
@@ -62,15 +67,18 @@ function NavBar() {
...
       </nav>
 
-      <BurgerMenu width={320}>
+      <BurgerMenu width={320} isOpen={isBurgerOpen} onStateChange={isMenuOpen}>
         <NavLink
           id="meetings"
           className="menu-item"
           activeClassName="bm-active"
           to="/campaigns"
           onClick={
```
Comment (darkweak) [inline frontend/src/components/NavBar.jsx:67]:
```suggestion
          onClick={() => {
            dispatch({ type: 'AMB_ALL' });
            dispatch({ type: 'ASS_ALL' });
            isBurgerOpenUpdate(false)
          }}
```

File: frontend/src/components/NavBar.jsx [hunk 4]
```diff
@@ -83,7 +91,10 @@ function NavBar() {
           to="/ambassadors"
           onClick={
             (() => dispatch({ type: "AMB_ALL" })) &&
-            (() => dispatch({ type: "ASS_ALL" }))
+            (() => dispatch({ type: "ASS_ALL" })) &&
```
Comment (darkweak) [inline frontend/src/components/NavBar.jsx]: Same here

File: frontend/src/components/styles/NavBar.scss [hunk 5]
```diff
@@ -68,7 +68,7 @@
   width: 36px;
   height: 30px;
   right: 20px;
-  top: 7px;
+  top: 27px;
```
Comment (darkweak) [inline frontend/src/components/styles/NavBar.scss:71]: Never use px, use rem instead

**Review by darkweak — CHANGES_REQUESTED** [review]

File: frontend/src/components/NavBar.jsx (hunk 3 above)
Comment (BastienSaulnier) [reply frontend/src/components/NavBar.jsx:67]: This will change today, anyway thank you :D

**Review by LoicBrassart — APPROVED** [review]
Your reducer would need a good refactoring,  but it's a low-priority task for now, we'll see if we can squeeze it before next week

**Review by LeoDiduch — APPROVED** [review]
SnE

**Pull request merged**